src/components/ha-button-menu.ts

```diff
@@ -97,6 +97,11 @@ export class HaButtonMenu extends LitElement {
     this._menu!.show();
   }
 
+  private _setTriggerAria() {
+    const triggerButton = this._triggerButton.shift();
+    if (triggerButton) triggerButton.ariaHasPopup = "menu";
```

Comment:
```suggestion
    if (triggerButton) {
    	triggerButton.ariaHasPopup = "menu";
    }
```

Reply:
Isn't that the same code?

Reply:
Yes. Coding style should be consistent. Not sure why prettier doesn't catch it any more